<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientClass</class>
 <widget class="QMainWindow" name="ClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="Swidgt">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_login">
       <widget class="QWidget" name="widget_4" native="true">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>140</y>
          <width>381</width>
          <height>331</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_title">
           <property name="font">
            <font>
             <family>仿宋</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>欢迎来到宠物小精灵pokemon~</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>用户名：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>15</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="line_username">
              <property name="maxLength">
               <number>16</number>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>密码：</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>30</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="line_password">
              <property name="maxLength">
               <number>16</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="button_login">
              <property name="text">
               <string>登录</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="button_register">
              <property name="text">
               <string>注册</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_menu">
       <widget class="QPushButton" name="Button_enter_battle">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>40</y>
          <width>241</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>战斗</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_query">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>170</y>
          <width>211</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>查询在线玩家及其精灵</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_logout">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>440</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_pokemons">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>290</y>
          <width>211</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>查看背包、仓库精灵</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_battle">
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>151</width>
          <height>181</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>150</y>
          <width>151</width>
          <height>181</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_return_menu1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>返回菜单</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_query">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>311</width>
          <height>481</height>
         </rect>
        </property>
        <property name="title">
         <string>当前在线玩家</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="list_player_online"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="Button_return_menu2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>返回菜单</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_fresh_online_player">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>100</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_pokemons">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>251</width>
          <height>431</height>
         </rect>
        </property>
        <property name="title">
         <string>背包</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="list_pokemon_bag"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="Button_pokemon_to_store">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>110</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>将精灵放到仓库-&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_pokemon_to_bag">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>150</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;-将精灵放到背包</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>80</y>
          <width>251</width>
          <height>431</height>
         </rect>
        </property>
        <property name="title">
         <string>仓库</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QListWidget" name="list_pokemon_store"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="Button_return_menu3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>返回菜单</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_test">
       <widget class="QPushButton" name="Button_socket">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>191</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>socket发送</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="Text_send">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>241</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="Text_read">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>250</y>
          <width>261</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="Text_status">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>110</y>
          <width>241</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_disconnect">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>110</y>
          <width>151</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>断开连接</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Client.qrc"/>
 </resources>
 <connections/>
</ui>
